parts/k8s/cloud-init/artifacts/cse_config.sh

```diff
@@ -416,6 +416,20 @@ ensureKubelet() {
   {{- end}}
   fi
 {{- end}}
+if [[ -n ${MASTER_NODE} ]]; then
+  wait_for_file 1200 1 /etc/systemd/system/apiserver-monitor.service || exit {{GetCSEErrorCode "ERR_FILE_WATCH_TIMEOUT"}}
+  systemctlEnableAndStart apiserver-monitor || exit {{GetCSEErrorCode "ERR_KUBELET_START_FAIL"}}
```

Comment:
Should we add an `ERR_K8S_API_SERVER_MONITOR_FAIL` for this? All the other uses I see for this error code are either starting `kubelet` or `kubelet-monitor`.

Reply:
Good question. Because (1) kubelet is a catch-all systemd service for ensuring all k8s components come online (including control plane components if it's a control plane node) and (2) we're in the `ensureKubelet` func, I think it makes sense to return this common error. Basically we're just adding to the set of required bootstrap conditions for launching the kubelet systemd service.

Reply:
👍🏻 Make sense to me.